worker-document-interface/src/main/java/com/hpe/caf/worker/document/model/Script.java

```diff
@@ -38,6 +38,11 @@
      */
     int getIndex();
 
+    /**
+     * Closes the bindings and context associated with this customization script. 
+     */
+    void closeBindings();
```

Comment:
Surely closing the bindings is entirely internal.  Why would we be exposing something like this on the external interface used by workers?